core/amber/src/main/python/core/util/print_writer/print_writer.py

```diff
@@ -0,0 +1,41 @@
+from datetime import datetime
+from typing import Iterator, List
+
+
+class SimpleBuffer:
+    def __init__(self, maximum_size=10, maximum_window=2):
+        self._max_size = maximum_size
+        self._max_interval = maximum_window
```

Comment:
add `_secs` in the variable name (or in comment)? so its easier to tell the time unit. Also 2 seconds seems too slow for me. I think we can do a few hundred miliseconds.

Reply:
added to be `max_flush_interval_in_ms` and `max_message_num`. changed to 500ms.